arches/app/models/resource.py

```diff
@@ -461,6 +468,8 @@ def index(self, context=None):
                         )
                         es_index.index_document(document=doc, id=doc_id)
 
+            resource_indexed.send(sender=self.__class__, instance=self)
```

Comment:
This is maybe out of scope, so happy to remove (I had not intentionally added it to this commit when reordering commits), but we use it to react to a potential change of permissions data (for example, to refresh a permissions cache entry), and that seems a more likely use-case with these changes.